README.md

```diff
@@ -96,25 +96,6 @@ npm run prod
 
 _Although the build of the client is production-ready, Nuxt's own HTTP server should never be exposed directly to the internet and a proper hosting tool like Nginx should always be used instead._
 
-### Server-Side Rendering (SSR)
-
-This client fully supports Server-Side Rendering. All the commands listed above have their SSR siblings:
-
-```bash
-npm run build:ssr
-npm run start:ssr
-npm run prod:ssr
-```
-
-#### Running standalone
```

Comment:
Is this provided somewhere else now?

Reply:
Standalone has nothing to do with CSR: https://nuxtjs.org/docs/configuration-glossary/configuration-build/#standalone

For serving the client, there's ``npm run prod``

Reply:
But is that included in the README after this new pull request or is it left for the users to figure out now?

Reply:
Just above this section, called "Running a  production build".